rosbag2_snapshot/src/snapshotter.cpp

```diff
@@ -30,7 +30,7 @@
 #include <rclcpp/rclcpp.hpp>
 #include <rosbag2_snapshot/snapshotter.hpp>
 
-#include <filesystem>
+#include <experimental/filesystem>
```

Comment:
@JWhitleyWork are you aware of best practices for how to handle this include path / namespace issue with std::filesystem vs std::experimental::filesystem on differing OS'es? I would like this to include whichever header is available at compile time, and then use the correct namespace, to make this code more cross-platform.